### PR TITLE
Adjusted tutorial based on additional comments in #15716

### PR DESCRIPTION
**Pull request by dariusztutaj**
**Description**
mTLS related tutorials adjusted to comments in #15716

### CODE REVIEW

File: docs/03-tutorials/00-api-exposure/apix-10-expose-and-secure-workload-with-certificate.md [hunk 1]
```diff
@@ -160,7 +162,7 @@ These calls return the code `200` response. If you call the service without the
   Call the secured Function
   </summary>
 
-Send a `GET` request with a token that has the `read` scope to the Function:
+Send a `GET` request to the Function with a token that has the `read` scope:
```
Comment (strekm) [inline docs/03-tutorials/00-api-exposure/apix-10-expose-and-secure-workload-with-certificate.md:165]: where's token? here you are sending req to the function with cert

**Review by strekm — CHANGES_REQUESTED** [review]

**Review by strekm — DISMISSED** [review]
/meow

File: docs/03-tutorials/00-api-exposure/apix-10-expose-and-secure-workload-with-certificate.md [hunk 2]
```diff
@@ -2,13 +2,15 @@
 title: Expose and secure a workload with a certificate
 ---
 
-This tutorial shows how to expose and secure a workload with mutual authentication using Kyma's mutual TLS Gateway.
+This tutorial shows how to expose and secure a workload with mutual authentication using mutual TLS Gateway.
```
Comment (NHingerl) [inline docs/03-tutorials/00-api-exposure/apix-10-expose-and-secure-workload-with-certificate.md:5]:
```suggestion
This tutorial shows how to expose and secure a workload with mutual authentication using a mutual TLS Gateway.
```
If you remove "Kyma's", you need an article

File: docs/03-tutorials/00-api-exposure/apix-10-expose-and-secure-workload-with-certificate.md [hunk 3]
```diff
@@ -2,13 +2,15 @@
...
 
 ## Prerequisites
 
 This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create one of those, follow the [Create a workload](./apix-01-create-workload.md) tutorial.
 
-Before you start, Set up [mTLS Gateway](../00-security/sec-03-setup-mtls-gateway.md) to allow mutual authentication in Kyma and make sure that you exported the [bundle certificates](../00-security/sec-03-setup-mtls-gateway#steps).
+Before you start, Set up:
```
Comment (NHingerl) [inline docs/03-tutorials/00-api-exposure/apix-10-expose-and-secure-workload-with-certificate.md:11]:
```suggestion
Before you start, set up:
```

File: docs/03-tutorials/00-api-exposure/apix-10-expose-and-secure-workload-with-certificate.md [hunk 4]
```diff
@@ -2,13 +2,15 @@
...
+- [Custom Domain](./apix-02-setup-custom-domain-for-workload.md) - skip step 5 (Create a Gateway CR)
```
Comment (NHingerl) [inline docs/03-tutorials/00-api-exposure/apix-10-expose-and-secure-workload-with-certificate.md:12]: if readers hop over to the Custom Domain document, will they remember to skip step 5? I would probably forget and blindly perform all steps. What happens if they perform step 5?

Consider adding a hint to that procedure that step 5 doesn't apply in certain (specified) cases.

Comment (dariusztutaj) [reply docs/03-tutorials/00-api-exposure/apix-10-expose-and-secure-workload-with-certificate.md:12]: Note added.

**Review by NHingerl — APPROVED** [review]

**Pull request merged**